marimo/_server/ai/providers.py

```diff
@@ -123,6 +136,368 @@ class ActiveToolCall:
     tool_call_args: str
 
 
+class PydanticGoogleProvider:
+    def __init__(self, model: str, config: AnyProviderConfig):
+        DependencyManager.pydantic_ai.require(why="for AI assistance")
+        DependencyManager.google_ai.require(
```

Comment:
there is `DependencyManager.require_many(` that will show a toast of two items missing:

for example
```python
DependencyManager.require_many(
            "to execute sql",
            DependencyManager.duckdb,
            DependencyManager.sqlglot,
        )
```